config.json

```diff
@@ -10,5 +10,6 @@
   "eventQueryTimeout": 20,
   "enableEtherFaucet": false,
   "enableRelayMetaTransaction": false,
-  "enableDeployIdentity": false
+  "enableDeployIdentity": false,
+  "fixedGasPrice": false
```

Comment:
This makes me think I could change it to `"fixedGasPrice": true`, which will not work and makes me wonder where I set the gas price. I would prefer the use of negative values here (which would mean ask via rpc) and rename it to "gasPrice".
Or something like
```
"gasPriceComputation": "rpc"   # or "fixed",
"gasPrice": 100
```

Reply:
(or not set the field at all, if you don't want a fixed gas price)

Reply:
not set the value at all will also work

Reply:
I think allowing a negative number to disable it, is similar to what we have now, but less explicit. I dont like if special numbers are treated special if you can solve it differently.

Reply:
I can change it to:
```
"gasPriceComputation": {
"method": "rpc",
"gasPrice": 100
}
```
But I think this is less self documenting, because how should I now know that the other option is named "fixed"

Reply:
Now I think someone would try true, this would fail with an error, and then she would probably just set it to a number.

Reply:
The gasPriceComputation map is fine.